gating/pre_merge_test/collect_logs.yml

```diff
@@ -0,0 +1,147 @@
+- hosts: "{{ target_hosts }}"
+  gather_facts: false
+  tasks:
+    - name: Ensure artefacts directories exists
+      file:
+        path: "{{ item }}"
+        state: directory
+      with_items:
+        - "{{ artifacts_dir }}/{{ inventory_hostname }}/host"
+        - "{{ artifacts_dir }}/{{ inventory_hostname }}/containers"
+      delegate_to: "localhost"
+
+    - name: Grab host data
+      command: >
```

Comment:
Is there are reason for not using the synchronize module ?

Reply:
Not really, I copied this exactly from:

https://github.com/rcbops/rpc-openstack/blob/newton/gating/pre_merge_test/collect_logs.yml

So if we want to make changes, I'd recommend we make them there and port the changes here.  They do the same actions so I'd like to keep them aligned if it all possible.